EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -699,9 +712,9 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
 	int ichan=(i.channelid()&0x3);
 	HcalElectronicsId eid(crate,slot,ifiber,ichan, false);
 	DetId did=emap.lookup(eid);
-	
+
 	if (!did.null()) { // unpack and store...
-	  if (did.det()==DetId::Calo && did.subdetId()==HcalZDCDetId::SubdetectorId) {
+	  if (HcalRawToDigi::unpackZDCQIE10_ == true && did.det()==DetId::Calo && did.subdetId()==HcalZDCDetId::SubdetectorId) {
```

Comment:
Is there another way you'd prefer? I followed examples that I could find - is there an alternate example I could implement?
I'll change the default to false.